common/src/main/resources/META-INF/openapi.json

```diff
@@ -1694,6 +1694,71 @@
                 "summary": "Search for artifacts",
                 "description": "Returns a paginated list of all artifacts that match the provided search criteria.\n"
             }
+        },
+        "/admin/logging/{logger}": {
+            "summary": "Manage a logger.",
+            "get": {
+                "responses": {
+                    "200": {
+                        "content": {
+                            "text/plain": {
+
+                            }
+                        },
+                        "description": "The current log level for this logger"
+                    }
+                },
+                "operationId": "getLogLevel",
+                "summary": "Returns the configured log level for the given logger"
+            },
+            "put": {
+                "parameters": [
+                    {
+                        "name": "level",
+                        "description": "The log level to configure",
+                        "schema": {
+                            "enum": [
+                                "DEBUG",
+                                "TRACE",
+                                "INFO",
+                                "FINE",
+                                "FINEST"
+                            ],
+                            "type": "string"
+                        },
+                        "in": "query",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "content": {
+                            "text/plain": {
+
+                            }
+                        },
+                        "description": "The configured log level"
+                    }
+                },
+                "operationId": "setLogLevel",
+                "summary": "Configures the log level for this logger."
+            },
+            "parameters": [
+                {
+                    "examples": {
+                        "Storage logger": {
+                            "value": "io.apicurio.registry.storage"
+                        }
+                    },
+                    "name": "logger",
+                    "description": "Name of the logger to manage",
+                    "schema": {
+                        "type": "string"
+                    },
+                    "in": "path",
+                    "required": true
+                }
+            ]
```

Comment:
I love that you updated the `openapi.json` file - but we should make the changes in Apicurio Studio first, then use Studio to publish the result to GH.  You have access to the API description in Studio already - it's called **Apicurio Registry API**